src/services/posts/fetch-posts.js

```diff
@@ -0,0 +1,36 @@
+const db = require("../../db")
+
+const fetchPosts = async(isFlagged) => {
+
+	
+	let selectClause = `
+	SELECT posts.id, posts."userId" , posts.title, posts.image,
+	posts.content,posts.published, posts."createdAt", posts.type,
+	posts."likesCount",
+	users."firstName", users."lastName", users.email,
+	users."profilePictureUrl" 
+	`
+
+	 const fromClause = `FROM posts`
+	  
+	 const joinClause = `INNER JOIN users on posts."userId" = users.id`
+	 let whereClause = ''
+	 let orderByClause = ''
```

Comment:
lets order by the most recently created by default

Reply:
sure

Reply:
i have a lot of tests failing when I decided to order it by the recently created posts by default. I will fix them

Reply:
a lot of tests is a stretch given the number of related tests but okay 😄

Reply:
Lool